src/PoCs/deleting.tsx

```diff
@@ -0,0 +1,9 @@
+import React, {VoidFunctionComponent } from "react";
+
+export const InstanceDeletion: VoidFunctionComponent  = () => {
+    return (
+        <div>
+            <p style={{ color: "#6A6E73" }}>Deleting</p>
+        </div>
+    );
+};
```

Comment:
The only question I have left is whether this color should also be moved to the CSS file and use the PF color variable instead.